packages/riipen-ui/src/components/Paper.jsx

```diff
@@ -0,0 +1,106 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+const Paper = props => {
+  const {
+    children,
+    classes,
+    component: Component,
+    elevation,
+    padding,
+    rounded,
+    variant,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const className=clsx(
+    "root",
+    classes,
+    rounded ? "rounded" : null,
+    variant === "outlined" ? "outlined" : "elevation"
+  );
+
+  return (
+    <React.Fragment>
+      <Component className={className} {...other} >
+        {children}
+      </Component>
+      <style jsx>{`
+        .root {
+          background-color: ${theme.palette.common.white};
+          padding: ${theme.spacing[padding]};
+          display: inline-block;  
```

Comment:
Can we make these in alphabetical order? We like to keep properties alphabetical but unfortunately there is not way to automate this with `styled-jsx` :( so we have to do it manually in the code review process.